extensions/amp-accordion/0.1/test/test-amp-accordion.js

```diff
@@ -18,14 +18,27 @@ import {Timer} from '../../../../src/timer';
 import {adopt} from '../../../../src/runtime';
 import {createIframePromise} from '../../../../testing/iframe';
 require('../amp-accordion');
+import * as sinon from 'sinon';
```

Comment:
This test didn't use sinon really, but there are a lot of files who are and not doing the import. I am going to send another PR to fix all the non-imported sinon. I think it depends on how the bundler bundles them but yeah I've seen a nasty test failure in `test-3p-frame` because sinon wasn't properly imported.